src/gui/sdl_mapper.cpp

```diff
@@ -550,6 +558,26 @@ class CJHatBind final : public CBind {
 
 bool autofire = false;
 
+static void set_joystick_led([[maybe_unused]] SDL_Joystick *joystick,
+                             [[maybe_unused]] const uint32_t lsb_color)
+{
+// Starting in SDL 2.0.14, basic LED support was added.
+#if SDL_VERSION_ATLEAST(2, 0, 14)
+	if (!joystick)
+		return;
+	if (!SDL_JoystickHasLED(joystick))
+		return;
+
+	// unpack the colors
+	const auto r = static_cast<uint8_t>((lsb_color & 0x00'ff'00'00) >> 16);
+	const auto g = static_cast<uint8_t>((lsb_color & 0x00'00'ff'00) >> 8);
+	const auto b = static_cast<uint8_t>((lsb_color & 0x00'00'00'ff) >> 0);
```

Comment:
If `lsb_color` were a struct, there would be no need to unpack it.

Reply:
Yup - we do have an `rbg24` struct (and I originally used it).

But then I thought we might want to set colors for some short period of time (and not block the Staging during those periods).  To do it, we'd use the `PIC_AddEvent` call, that takes in a function_pointer, delay, and a uint32 argument.  

It would let us turn on the LED for second, then shut it off (using a couple PIC_AddEvent calls).  That said: I'm not doing in this PR, so maybe I should just use RGB for now?

Reply:
OK - `rgb24` is in!